Gaijinizer patch/Map447.txt

```diff
@@ -3,446 +3,570 @@
-
+> CONTEXT: Map447/events/1/pages/0/18/Dialogue
+When my hazy consciousness came back to me, I saw my
+rescuer standing in front of me. A demon dressed in risque
+Japanese clothing.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 ……きみ、大丈夫？
-> CONTEXT: Map447/events/1/pages/0/23/Dialogue/enemy15 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/23/Dialogue/enemy15
+Hey, are you okay?#Ron
 > END STRING
 
 > BEGIN STRING
 き、きみは……！？
-> CONTEXT: Map447/events/1/pages/0/25/Dialogue/kazuya-2 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/25/Dialogue/kazuya-2
+W-Who are you?!#Ron
 > END STRING
 
 > BEGIN STRING
 いやぁ、もうダメかと思ったけど……
 
 なんとか合流が間に合ったみたいだね。
-> CONTEXT: Map447/events/1/pages/0/27/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/27/Dialogue/sirousagi_fc1
+Phew, I thought he was done for there... but it
+looks like you got here just in the nick of time.#Ron
 > END STRING
 
 > BEGIN STRING
 悪魔ながらに、人間を守る立場にある『統治者』……
 
 ネコマタの\\C[2]『夜深（やみ）』\\C[0]ちゃん！
-> CONTEXT: Map447/events/1/pages/0/31/Dialogue/sirousagi_fc1 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/31/Dialogue/sirousagi_fc1
+Though she may be a demon, she is the "Sovereign"
+who protects humanity, the nekomata \\C[2]Yami\\C[0]-chan!#Ron
 > END STRING
 
 > BEGIN STRING
 な、何なの！？貴女は…！
 
 人が優雅に食事している最中に……！
-> CONTEXT: Map447/events/1/pages/0/39/Dialogue/highslag_fc1 < UNTRANSLATED
+> CONTEXT: Map447/events/1/pages/0/39/Dialogue/highslag_fc1
+W-What are you doing?!
 
+You don't interrupt someone who's gracefully
+consuming a human...!#RonLocalization
 > END STRING
 
 > BEGIN STRING
 なーにが『優雅に』よ！
 
 神社の外観、こんなにぶっ壊しておいて！！
-> CONTEXT: Map447/events/1/pages/0/47/Dialogue/enemy30 < UNTRANSLATED
+> CONTEXT: Map447/events/1/pages/0/47/Dialogue/enemy30
+What was graceful about that?!
 
+You've blown the outside of the shrine to
+smithereens!!#Ron
 > END STRING
 
 > BEGIN STRING
 すさまじい物音と、邪気を感じたから来てみたら……！
 
 私の神社で、好き勝手なことはさせないんだからっ！！
-> CONTEXT: Map447/events/1/pages/0/51/Dialogue/enemy30 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/51/Dialogue/enemy30
+I'm here because I heard a huge noise and sensed
+evil! I'm not gonna let you do whatever you want
+in my shrine!!#Ron
 > END STRING
 
 > BEGIN STRING
 はぁ……
 人間のオス一匹、この力でどうともなると思ってたけど……
 
 確かに、このままだと分が悪いわね……
-> CONTEXT: Map447/events/1/pages/0/67/Dialogue/highslag_fc1 < UNTRANSLATED
-
+> CONTEXT: Map447/events/1/pages/0/67/Dialogue/highslag_fc1
+Haah... I thought this power would let me have
+my way with a single human male. But the way
+things are going, it seems my chances of victory
+are slim.#Ron
 > END STRING
 
 > BEGIN STRING
 いいわ、今は退いてあげる。
 
 貴方に『その気がある』って事が分かった以上
 戦い方を改める必要がありそうだし……
-> CONTEXT: Map447/events/1/pages/0/72/Dialogue/highslag_fc1 < UNTRANSLATED
+> CONTEXT: Map447/events/1/pages/0/72/Dialogue/highslag_fc1
+That's fine, I'll withdraw for now.
 
+It would seem I need to revise my fighting style
+now that I know you have that sort of interest.#Ron
 > END STRING
```

Comment:
Minor revision that I think sounds a bit better:
```
That's fine, I'll withdraw for now.

It would seem I need to revise my fighting style
now that I know you're interested in "that"...
```